charts/rancher-monitoring/v0.0.2/values.yaml

```diff
@@ -293,38 +283,27 @@ prometheus:
   level: cluster
   auth:
     args:
-    - --proxy-url
-    - http://localhost:9090
-    - --listen-address
-    - $(POD_IP):9090
-    - --filter-reader-labels
-    - prometheus
-    - --filter-reader-labels
-    - prometheus_replica
+    - --proxy-url=http://localhost:9090
+    - --listen-address=$(POD_IP):9090
+    - --filter-reader-labels=prometheus
+    - --filter-reader-labels=prometheus_replica
     env:
     - name: POD_IP
       valueFrom:
         fieldRef:
           fieldPath: status.podIP
   apiGroup: "monitoring.coreos.com"
   image:
-    repository: prom/prometheus
+    repository: rancher/prometheus
```

Comment:
I think we already pushed this image to rancher repo but I really want to follow the pattern of what we do for alert manager `image = strings.Replace(image, "prom/", "rancher/prom-", 1)` in types https://github.com/rancher/types/blob/master/image/mirror.go#L21

Reply:
https://github.com/rancher/system-charts/pull/15#issuecomment-465339315